custom/conf/app.example.ini

```diff
@@ -237,10 +237,13 @@ RUN_MODE = ; prod
 ;; PPROF_DATA_PATH, use an absolute path when you start gitea as service
 ;PPROF_DATA_PATH = data/tmp/pprof
 ;;
-;; Landing page, can be "home", "explore", "organizations" or "login"
+;; Landing page, can be "home", "explore", "organizations", "login", or "custom"
 ;; The "login" choice is not a security measure but just a UI flow change, use REQUIRE_SIGNIN_VIEW to force users to log in.
 ;LANDING_PAGE = home
```

Comment:
That is one approach.
Another would have been to automatically use an unrecognized value as the landing page URL.
I'm fine with both approaches.

Reply:
We should avoid introducing more setting options here. Indeed we can reuse the `LANDING_PAGE`

`LANDING_PAGE=home`:  builtin home URL
`LANDING_PAGE=/org/repo`: the `/org/repo` URL if the value starts with a slash.

Reply:
Or `LANDING_PAGE= https://anotherwebsite`

Reply:
I've updated it to just be the one option now that covers all cases as discussed above